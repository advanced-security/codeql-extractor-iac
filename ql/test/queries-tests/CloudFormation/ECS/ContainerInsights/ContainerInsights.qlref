security/CloudFormation/ECS/ContainerInsights.ql
