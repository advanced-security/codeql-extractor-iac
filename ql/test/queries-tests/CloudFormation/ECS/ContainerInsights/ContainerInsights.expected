| ecs.yml:41:5:47:2 | CloudFormation ECS Cluster | ECS Cluster should have cluster settings enabled |